app/event-publisher/index.js

```diff
@@ -0,0 +1,51 @@
+const { PublishEventBatch } = require('ffc-ahwr-event-publisher')
+const config = require('../config')
+
+const raise = async (event) => {
+  await new PublishEventBatch(config.eventQueue).sendEvents([
+    {
+      name: 'application-status-event',
+      properties: {
+        id: `${event.application.id}`,
+        sbi: `${event.application.data.organisation.sbi}`,
+        cph: 'n/a',
```

Comment:
can we remove this?

Reply:
Sadly not as these are all required fields within the event publisher event schema. See here - https://github.com/DEFRA/ffc-ahwr-event-publisher/blob/main/app/event-publisher/event-schema.js

Reply:
NP - one ticket to pickup when possible 

https://dev.azure.com/defragovuk/DEFRA-FFC/_search?text=backwards*&type=workitem&lp=workitems-Project&filters=Projects%7BDEFRA-FFC%7D&pageSize=25


We could make CPH optional going forwards as it is no longer sent